GameServerLib/GameObjects/AttackableUnits/AI/ObjAIBase.cs

```diff
@@ -59,7 +59,9 @@ public class ObjAiBase : AttackableUnit, IObjAiBase
         {
             _itemManager = game.ItemManager;
             _scriptEngine = game.ScriptEngine;
-            CharData = _game.Config.ContentManager.GetCharData(Model);
+
+            CharData = (CharData) _game.Config.ContentManager.GetCharData(Model);
```

Comment:
Use ICharData.
```suggestion
            CharData = (ICharData) _game.Config.ContentManager.GetCharData(Model);
```
**Requires**:
```
        public ICharData CharData { get; } // line 38 ObjAIBase.cs
```
```
        public void LoadStats(ICharData charData) // line 109 Stats.cs
```